scripts/lib/CIME/utils.py

```diff
@@ -26,7 +26,6 @@ def expect(condition, error_msg, exc_type=SystemExit, error_prefix="ERROR:"):
         if logger.isEnabledFor(logging.DEBUG):
             import pdb
             pdb.set_trace()
-        logger.error("%s %s", error_prefix,error_msg, exc_info=True)
```

Comment:
Yes, it was added in a previous pr today, but I didn't like the default behavior.